changelogs/unreleased/gh-6063-net-box-ping-args.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/Netbox
+
+- Fixed a bug when conn:ping() returned false if it was invoked with
```

Comment:
Lets be consistent with all the other changelogs and use `*` here instead of `-` as a line starter.

Reply:
My bad. Fixed it